src/System.Runtime.Serialization.Json/pkg/System.Runtime.Serialization.Json.pkgproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Version>4.0.2.0</Version>
+    <Version>4.0.3</Version>
```

Comment:
Well, there are also instances where version is specified with 3 parts. and since that's more accurate I decided to keep it as is.